tools/create_f2fs_image.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+SYZDIR=/usr/local/google/home/jiahengh/gopath/src/github.com/google/syzkaller
+declare -a Op1=("-a 0 " "-a 1 " "")
+declare -a Op2=("-s 1 " "-s 2 " "-s 3" "")
+declare -a Op3=("-m " "")
```

Comment:
Nit:  The spaces within the quotes are unnecessary

Reply:
I removed the spaces in the command instead but the effect should be the same

Reply:
Effect may be the same, but it's easier to read the commands when the flags have spaces between them.